lib/iris/_lazy_data.py

```diff
@@ -97,7 +97,10 @@ def as_concrete_data(data, **kwargs):
 
     """
     if is_lazy_data(data):
-        # Realise dask array.
+        # Realise dask array, ensuring the data result is always a NumPy array.
+        # In some cases dask may return a scalar numpy.int/numpy.float object
+        # rather than a numpy.ndarray object.
+        # Recorded in https://github.com/dask/dask/issues/2111.
```

Comment:
This comment should probably be moved now that we're not ensuring `data` is a NumPy array here.